src/auth/auth.service.ts

```diff
@@ -22,13 +25,53 @@ export class AuthService {
   }
 
   async login(user: any) {
-    const payload = { email: user.email, sub: user.userId };
-    return {
-      access_token: this.jwtService.sign(payload),
-    };
+    const tokens = await this.getTokens(user.userId, user.email);    
+    await this.usersService.updateRefreshTokens(user.email, tokens.refresh_token);
+    return tokens
   }
 
   async signUp(user: SignUpDto) {
     return await this.usersService.signUp(user);
   }
+
+  async getTokens(userId: number, email: string): Promise<Tokens> {
+    const jwtPayload: JwtPayload = {
+      sub: userId,
+      email: email,
+    };
+
+    const [accessToken, refreshToken] = await Promise.all([
+      this.jwtService.signAsync(jwtPayload, {
+        secret: this.config.get<string>('JWT_SECRET'),
+        expiresIn: this.config.get<string>('JWT_EXPIRES_IN'),
+      }),
+      this.jwtService.signAsync(jwtPayload, {
+        secret: this.config.get<string>('REFRESH_SECRET'),
```

Comment:
Add this new env var validation in src/env.validation.ts

Reply:
added